app/our-business/our-product/page.tsx

```diff
@@ -14,7 +23,49 @@ export default function OurProduct() {
       />
       {/* <-- === Breadcrumb End === --> */}
 
-      {/* <-- === The content must be placed right here === --> */}
+      <main className="lg:px-content-padding-lg 2xl:px-content-padding-2xl pt-content-padding-top-sm lg:pt-content-padding-top-lg 2xl:pt-content-padding-top-2xl mb-32 lg:mb-48">
+        <section className="flex flex-col w-full gap-5 lg:gap-7 lg:items-center px-content-padding-sm">
+          <h2 className="text-[48px] lg:text-6xl font-aeonik-medium text-primary">
+            Our Product
+          </h2>
+          <div className="block">
+            <p className="text-[15px] lg:text-[17px] lg:font-light text-charcoal leading-relaxed lg:text-center">
+              Arthawena has expanded its product range to include various types
+              of cans, <br className="hidden lg:block" />
+              custom-tailored to meet the technical specifications and design
+              preferences of our <br className="hidden lg:block" />
+              customers. Our sales team is extensively trained and equipped with
+              comprehensive <br className="hidden lg:block" />
+              technical knowledge to advise customers in selecting the most
+              suitable <br className="hidden lg:block" />
+              specifications according to their requirements.
+            </p>
+          </div>
+        </section>
+
+        <section className="w-screen lg:w-full lg:px-14 flex flex-col gap-2 lg:gap-4 mt-[74px] lg:mt-32">
+          {productData.map((product) => (
+            <div
+              key={`product-${product.title}`}
+              className="aspect-square lg:aspect-auto block relative w-full h-full lg:h-auto object-cover object-center"
+            >
+              <Image
```

Comment:
should we set quality for the image to 100 ?

Reply:
will do, sir!